docs/greptimecloud/integrations/otlp.md

```diff
@@ -17,5 +21,26 @@ The request uses binary protobuf to encode the payload, so you need to use packa
 The package names may change according to OpenTelemetry, so we recommend that you refer to the official OpenTelemetry documentation for the most up-to-date information.
 :::
 
-For more information about the OpenTelemetry SDK, please refer to the official documentation for your preferred programming language.
-
+## OpenTelemetry Collector
```

Comment:
Duplicate content leads to additional maintenance cost. We may support a component in the future to solve this problem. @sunchanglong